lib/slather/profdata_coverage_file.rb

```diff
@@ -246,16 +246,14 @@ def supported_file_extensions
     private :supported_file_extensions
 
     def ignored?
-      # This indicates a llvm-cov coverage warning (occurs if a passed in source file 
-      # is not covered or with ccache in some cases).
-      ignore = source_file_pathname.to_s.end_with? "isn't covered."
+      path = source_file_pathname.to_s
 
-      if !ignore
-        # Ignore source files inside of platform SDKs
-        ignore = (/Xcode.*\.app\/Contents\/Developer\/Platforms/ =~ source_file_pathname.to_s) != nil
-      end
+      return true if path.end_with?("isn't covered.")
+      return true if path.include?("/Xcode.*\.app\/Contents\/Developer\/Platforms")
+      return true if path.include?("/SourcePackages/checkouts")
```

Comment:
Do we want to add these two new paths here? If the user adds something to the include list, will that override this ignore?
Ignoring upstream dependencies should probably be up to the developer adding them to the ignore list and not a default here since many different dependency managers exist and will do this in different ways in different folders.

Reply:
That's a good point. I debated with myself over this. I have no problem removing them.

No, ignore overrides source-files. My thought being that it's probably more desirable to be able to define the source-file glob(s) in the config and then winnow out anything that you want ignored.

Reply:
@jarrodlombardo-EventBase I did update this to remove the exclusion of `/SourcePackages/checkouts`, though I left the platform exclusion in place. I can't imagine a scenario in which someone would want to slather something `Platforms`.

I can remove that as well if you really want, but I think the expected behavior that most people would assume is that SDK files are not slathered.